source/client-side-operations-timeout/etc/templates/deprecated-options.yml.template

```diff
@@ -0,0 +1,233 @@
+description: "operations error if timeoutMS is used with a deprecated timeout option"
+
+schemaVersion: "1.2"
+
+initialData:
+  - collectionName: &collectionName coll
+    databaseName: &databaseName test
```

Comment:
Because this file doesn't use a file-level `createEntities` directive, the db/collection entities for these aren't declared until each test. I used YAML anchors here so the tests that need these entities can reference them by anchors instead of re-declaring the names everywhere.

Reply:
Gotcha